src/app/api/gql/dataset-history.graphql

```diff
@@ -0,0 +1,70 @@
+query getDatasetHistory($datasetId: DatasetID!, $perPage: Int, $page: Int) {
+  datasets {
+    byId(datasetId: $datasetId) {
+      id
+      owner {
+        id
+        name
+      }
+      name
+      metadata {
+        currentSchema(format: PARQUET_JSON) {
+          content
```

Comment:
Why do we need schema for the "history" tab?

Reply:
yes. Perhaps no need for now.